src/main/java/org/haffson/adventofcode/utils/ProblemStatus.java

```diff
@@ -0,0 +1,21 @@
+package org.haffson.adventofcode.utils;
+
+import org.haffson.adventofcode.ProblemStatusEnum;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ProblemStatus {
+
+    private ProblemStatus() {
+    }
+
+    public static Map<Integer, ProblemStatusEnum> getProblemStatusMap(int part1, int part2,
+                                                                      ProblemStatusEnum status1, ProblemStatusEnum status2) {
+        final Map<Integer, ProblemStatusEnum> problemStatus = new HashMap<>();
+
+        problemStatus.put(part1, status1);
+        problemStatus.put(part2, status2);
+        return problemStatus;
```

Comment:
Matthias hatte das glaube ich schon mal angemerkt. Du möchtest den Problem-Status ja nicht mehr im laufenden Programm endern, weshalb deine Map unmodifiable sein kann. Da kannst du einfach folgende Methode nutzen und direkt returnen:
```suggestion
        return Map.of(
            part1, status1,
            part2, status2
            );
```

Reply:
Hallo Jan,
danke für Deine hilfreichen Kommentare. Leider gibt bei mir Map.of immer "Cannot resolve method". Ich glaube, ich benutze bei AoC Java 8 (ich bin mir aber nicht sicher). Und bei Java 8 gibt es (ich bin mir wieder nicht sicher :-D) kein Map.of...

Reply:
Tatsächlich! Das kannst du in deiner `build.gradle` sehen. Dort steht

```
sourceCompatibility = 1.8
targetCompatibility = 1.8
```

Diesen Monat ist sogar schon Java 17 herausgekommen. Als optionale Aufgabe kannst du nach Abschluss dieses PR vielleicht ein Update vornehmen. Die Schritte wären:
* Version in der `build.gradle` erhohen
* Version in den Project Settings von IntelliJ anpassen (`Strg`+`Shift`+`Alt`+`S`)
* Dein Code wird dann eventuell ein paar Fehler haben, die aber relativ leicht zu beheben sein sollten.

Aber wie gesagt: optional. Aber dann lernst du wenigstens auf dem aktuellen Stand. 👍

Reply:
Sehr cool! Ja, das werde ich gerne machen. Hatte auch schon danach geguckt, wie man das am besten anstellt. Danke für die Anleitung.

Reply:
Keine Gewähr für Vollständigkeit. 😅